modules/core/04-channel/keeper/upgrade.go

```diff
@@ -81,9 +80,8 @@ func (k Keeper) ChanUpgradeTry(
 		return types.Upgrade{}, errorsmod.Wrapf(types.ErrChannelNotFound, "port ID (%s) channel ID (%s)", portID, channelID)
 	}
 
-	// the channel state must be in OPEN or INITUPGRADE if we are in a crossing hellos situation
-	if !collections.Contains(channel.State, []types.State{types.OPEN, types.INITUPGRADE}) {
-		return types.Upgrade{}, errorsmod.Wrapf(types.ErrInvalidChannelState, "expected one of [%s, %s], got %s", types.OPEN, types.INITUPGRADE, channel.State)
+	if channel.State != types.OPEN {
```

Comment:
we've got the sparkling `IsOpen` we can use now :sparkles:

Reply:
I actually find it easier to read like this 😅  but we can change it too